Makefile

```diff
@@ -8,11 +8,17 @@ lint:
 	go get github.com/golang/lint/golint
 	golint *.go
 
-test: sysfs/fixtures/.unpacked
+test: sysfs/fixtures/.copied
```

Comment:
I have a renaming routine quite similar to the copy code in this PR. Obviously, this means that after a make test, every renamed file will show up as missing for git. Also, giving up the marker makes the test process slower. We have to walk the file try every time to check for files that need renaming. And a file removed from git won't get the renamed version removed automatically. For that, we'd have to rebuild the directory from scratch every time (also doable if that is your preference).